database/resources/project-ideas.json

```diff
@@ -68,5 +68,12 @@
     "url": "https://www.interviewbit.com/blog/blockchain-projects/",
     "category": "resources",
     "subcategory": "project-ideas"
+  },
+    {
+    "name": "Java Projects",
+    "description": "Are you beginner looking for interesting Java Projects ideas? If yes, check out this article! It will show you 100+ Java project ideas and topics with source code for coders like you.",
```

Comment:
```suggestion
    "description": "Are you a beginner and looking for interesting Java Projects ideas? If yes, check out this article! It will show you 100+ Java project ideas and topics with source code for coders like you.",
```